core/domain/feedback_services.py

```diff
@@ -366,6 +366,30 @@ def _get_thread_from_model(thread_model):
         thread_model.created_on, thread_model.last_updated)
 
 
+def get_new_messages_for_exp_id(exp_id):
+    """Fetch the feedback threads for the given exploration, and use those to
+    finally get the new (open) feedback messages.
```

Comment:
Ah, I see. For that sort of thing, then, the standard way to do it is to use 'open' everywhere except the final point in the UI, at which you change it to 'new'. That way you don't have multiple words in the backend code that mean the same thing.